extensions/adobe/experience/decisioning/decisionevent.schema.json

```diff
@@ -20,7 +20,56 @@
   "description": "A decision event is used to capture observations about the outcome and context of a decision activity. The decision event captures information about how the decision made, when it was occurring, what options were proposed (chosen) and what contextual state existed that either influenced the decision or could be observed during the decision process. The decision event also captures the proposition ID, a globally unique identifier that can be used to correlate the decision to other events. DecisionEvents are not only relatable to Experience Events that impacted a decision but also to ExperienceEvents that are a direct response to a proposition. It is the expectation that applications reference the proposition ID in every experience event that was influenced by the propositions. A user profile's proposition-response history is maintained with these proposition IDs.",
   "definitions": {
     "decisionevent": {
-      "properties": {}
+      "properties": {
+        "https://ns.adobe.com/experience/decisioning/propositions": {
+          "title": "Proposition Details",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/proposition-details"
+          },
+          "description": "Details about the results of the decision. An array providing the output of a single decision event, one proposition per activity parameter in the request."
+        },
```

Comment:
Align Decision event property name with name of Experience Event - Proposition Interaction Mix-in